app/src/main/java/com/pileproject/drive/programming/visual/block/selection/IfMachineIsOutOfLineBlock.java

```diff
@@ -18,32 +18,30 @@
 import android.content.Context;
 
 import com.pileproject.drive.R;
+import com.pileproject.drive.execution.CarControllerBase;
 import com.pileproject.drive.execution.MachineController;
-import com.pileproject.drive.execution.NxtController;
 import com.pileproject.drive.preferences.BlockPreferences;
 
 /**
  * This block check the light sensor's value
- * If the value is higher than threshold, it means nxt is on the
+ * If the value is higher than threshold, it means the machine is on the
  * light-colored floor and out of black line.
  *
  * @author <a href="mailto:tatsuyaw0c@gmail.com">Tatsuya Iwanari</a>
  * @version 1.0 7-July-2013
  */
-public class IfNxtIsOutOfLineBlock extends SelectionBlock {
+public class IfMachineIsOutOfLineBlock extends SelectionBlock {
 
     private int mThreshold;
 
-    public IfNxtIsOutOfLineBlock(Context context) {
-        super(context, R.layout.block_if_nxt_is_out_of_line);
+    public IfMachineIsOutOfLineBlock(Context context) {
+        super(context, R.layout.block_if_machine_is_out_of_line);
 
         mThreshold = BlockPreferences.get(context).getLineSensorThreshold();
     }
 
     @Override
     protected boolean evaluateCondition(MachineController controller) {
-        // comment is weird.
-        // getLightPercent returns tenfold value
-        return ((NxtController) controller).getLineSensorValue() > mThreshold * 10;
```

Comment:
Oh, they were fixed in [here](https://github.com/PileProject/drivecommand/blob/develop/src/main/java/com/pileproject/drivecommand/model/nxt/NxtProtocol.java#L93). I missed it.